README.md

```diff
@@ -41,96 +72,186 @@ pod "KlaviyoSwift"
 
-Prior to version 1.7.0, the Klaviyo SDK used the [Apple identifier for vendor (IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) to facilitate anonymous tracking. Starting with version 1.7.0, the SDK will use a cached UUID that is generated when the SDK is initialized. For existing anonymous profiles using IDFV, the SDK will continue to use IDFV, instead of generating a new UUID.
+### Collecting Push Tokens
 
-## Profile properties and Identifiers
+In order to send push notifications to your users, you must collect their push tokens and register them with Klaviyo.
+This is done via the `KlaviyoSDK().set(pushToken:)` method, which registers a push token and current authorization state
+via the [Create Client Push Token API](https://developers.klaviyo.com/en/reference/create_client_push_token).
 
-Whenever an email (or other identifier) is provided to use via our APIs we will retain that information for future calls so that new data is tracked against the right profile. However, you are always free to override the current identifiers by passing new ones into a customer properties dictionary.
+* Call [`registerForRemoteNotifications()`](https://developer.apple.com/documentation/uikit/uiapplication/1623078-registerforremotenotifications)
+to request a push token from APNs. This is typically done in the [`application:didFinishLaunchingWithOptions:`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application) method of your app delegate.
+* Implement the delegate method [`application:didRegisterForRemoteNotificationsWithDeviceToken`](https://developer.apple.com/documentation/appkit/nsapplicationdelegate/1428766-application)
+in your application delegate to receive the push token from APNs and register it with Klaviyo.
 
+Below is the code to do both of the above steps:
 ```swift
-KlaviyoSDK().set(email: "junior@blob.com")
-```
+import KlaviyoSwift
 
-## Push Notifications
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    KlaviyoSDK().initialize(with: "YOUR_KLAVIYO_PUBLIC_API_KEY")
 
-Implementing push notifications requires a few additional snippets of code to enable.:
+    UIApplication.shared.registerForRemoteNotifications()
 
-1. Registering users for push notifications.
-2. Sending resulting push tokens to Klaviyo.
-3. Handling when users attempt to open your push notifications.
+    return true
+}
+
+func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
+    KlaviyoSDK().set(pushToken: deviceToken)
+}
+```
+#### Push Tokens And Multiple Profiles
+
+Klaviyo SDK will disassociate the device push token from the current profile whenever it is reset by calling
+`set(profile:)` or `resetProfile`. You should call `set(pushToken:)` again after resetting the currently tracked profile
```

Comment:
This is no longer true.

Reply:
This should say the sdk will automatically associate the new token with the new profile.

Reply:
Would it make sense to simplify this section to something like the following?

Whenever you call 'set(Profile)' or 'resetProfile', the KlaviyoSDK will automatically remove the token from the previous profile and move it to the new profile. This process prevents the device from receiving push notifications intended for the old profile.

Reply:
Based on the internal discussion we decided to remove this section and add the comments to the API. Seemed like initialize, resetProfile already had the comments. Added a comment to the set profile.